docs-src/cloud-context/namespaces-delete.md

```diff
@@ -0,0 +1,29 @@
+---
+id: namespaces-delete
+title: How to delete a Namespace in Temporal Cloud
+sidebar_label: Delete a Namespace
+description: To delete a Namespace in Temporal Cloud, use either Temporal Cloud UI or tcld.
+tags:
+  - guide-context
+---
+
+:::info
+
+The user who creates a [Namespace](/namespaces) is automatically granted [Namespace Admin](/cloud/#namespace-level-permissions) permission for that Namespace.
+
+To delete a Namespace, a user must have the Developer or Global Admin account-level [Role](/cloud/#account-level-roles).
+
+:::
+
+### Delete a Namespace using Temporal Cloud UI
+
+1. Go to the Temporal Cloud UI and log in.
+2. On the left side of the window, click **Namespaces**.
+3. On the **Namespaces** page, select the Namespace Name you have Developer or Global Admin permission for.
+4. Click **Edit** in the upper-right portion of the window.
+5. In the text box provided, type `DELETE` to confirm the deletion of that Namespace.
+6. Click **Delete**.
```

Comment:
```suggestion
1. On the left side of the window, select **Namespaces**.
1. On the **Namespaces** page, select a Namespace Name.
1. On the Namespace page, select **Edit** in the upper-right portion of the window.
1. On the **Edit** Namespace page, select **Delete Namespace** in the upper-right portion of the window.
1. In the **Delete Namespace** dialog, type `DELETE` to confirm the deletion of that Namespace.
1. Select **Delete**.
```
Changes:
- Added step to select **Delete Namespace** button.
- Added name of **Delete Namespace** dialog.
- Replaced "click" with "select".
- Simplified numbering of ordered list.

The last two items will become an official part of our style guide Real Soon Now (TW-358).